app/models/on_the_fly_calc.rb

```diff
@@ -60,11 +61,11 @@ def scores(questions)
         calculate_assessment
       else
         assessments = ReviewResponseMap.get_assessments_for(team)
-        score_team[:scores] = Answer.compute_scores(assessments, questions[:review])
+        score_team[:scores] = Answer.compute_scores(assessments, questions)
```

Comment:
I dont think this modification is correct. Please check the `compute_scores` method in `Answer` model.